test/model/test_quantize_mlp.py

```diff
@@ -204,3 +205,103 @@ def test_quantize_mlp_weights_only_optimizers(weights, optimizer, frozen, device
 def test_quantize_mlp_wrong_optimizer(weights, optimizer, device):
     with pytest.raises(ValueError):
         _test_quantize_mlp(weights, None, optimizer, False, device)
+
+
+def _test_requantize_mlp(weights, activations, optimizer, frozen, device, serialization):
```

Comment:
As explained in the other pull-request, please create a single test_requantize_mlp.py file with just the serialize test (no need to compare with a float model since it is done in the quantize test already).